x/auth/client/cli/query.go

```diff
@@ -327,6 +329,77 @@ $ %s query txs --%s 'message.sender=cosmos1...&message.action=withdraw_delegator
 	return cmd
 }
 
+// QueryBlocksByEventsCmd returns a command to search through blocks by events.
+func QueryBlocksByEventsCmd() *cobra.Command {
```

Comment:
any chance we can move this to where the other tendermint commands live in the repo? https://github.com/cosmos/cosmos-sdk/blob/3676b2f2e70e54062eada1719ccbb7c2429fc966/server/tm_cmds.go#L1

Would be nice to not house non account related things outside of auth.

Reply:
I'm not seeing a `server/tm_cmds.go` in `main` right now?

Reply:
yep will do